tracer/src/Datadog.Trace/ClrProfiler/InstrumentationDefinitions.cs

```diff
@@ -80,6 +80,17 @@ internal static TraceMethodPayload GetTraceMethodDefinitions()
             };
         }
 
+        internal static TraceMethodPayload GetProfilerDefinitions()
+        {
+            return new TraceMethodPayload
+            {
+                // Fixed Id for definitions payload (to avoid loading same integrations from multiple AppDomains)
+                DefinitionsId = "A11B1AD49BF341C7810983E3AA82D409",
```

Comment:
Where did this ID come from?

Reply:
As the rest of the Ids - a generated, unique, UUID